packages/react-core/src/components/Form/__tests__/__snapshots__/FormGroup.test.tsx.snap

```diff
@@ -1,679 +1,35 @@
-        Validated FormGroup
-      </div>
-    </div>
-  </div>
-</FormGroup>
-`;
+exports[`FormGroup should render form group validated success variant 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"label-id\\"><span class=\\"pf-c-form__label-text\\">label</span></label> </div><div class=\\"pf-c-form__group-control\\"><input id=\\"id\\"><div class=\\"pf-c-form__helper-text pf-m-success\\" id=\\"label-id-helper\\" aria-live=\\"polite\\">Validated FormGroup</div></div></div>"`;
 
-exports[`FormGroup component should render form group validated warning variant 1`] = `
-<FormGroup
-  fieldId="label-id"
-  helperText="Validated FormGroup"
-  label="label"
-  validated="warning"
->
-  <div
-    className="pf-c-form__group"
-  >
-    <div
-      className="pf-c-form__group-label"
-    >
-      <label
-        className="pf-c-form__label"
-        htmlFor="label-id"
-      >
-        <span
-          className="pf-c-form__label-text"
-        >
-          label
-        </span>
-      </label>
-       
-    </div>
-    <div
-      className="pf-c-form__group-control"
-    >
-      <input
-        id="id"
-      />
-      <div
-        aria-live="polite"
-        className="pf-c-form__helper-text pf-m-warning"
-        id="label-id-helper"
-      >
-        Validated FormGroup
-      </div>
-    </div>
-  </div>
-</FormGroup>
-`;
+exports[`FormGroup should render form group validated warning variant 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"label-id\\"><span class=\\"pf-c-form__label-text\\">label</span></label> </div><div class=\\"pf-c-form__group-control\\"><input id=\\"id\\"><div class=\\"pf-c-form__helper-text pf-m-warning\\" id=\\"label-id-helper\\" aria-live=\\"polite\\">Validated FormGroup</div></div></div>"`;
 
-exports[`FormGroup component should render form group variant with function helperText 1`] = `
-<div>
-  <div
-    class="pf-c-form__group"
-  >
-    <div
-      class="pf-c-form__group-label"
-    >
-      <label
-        class="pf-c-form__label"
-        for="label-id"
-      >
-        <span
-          class="pf-c-form__label-text"
-        >
-          Label
-        </span>
-      </label>
-       
-    </div>
-    <div
-      class="pf-c-form__group-control"
-    >
-      <input
-        id="label-id"
-      />
-      <div>
-        label
-      </div>
-    </div>
-  </div>
-</div>
-`;
+exports[`FormGroup should render form group variant with function helperText 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"label-id\\"><span class=\\"pf-c-form__label-text\\">Label</span></label> </div><div class=\\"pf-c-form__group-control\\"><input id=\\"label-id\\"><div>label</div></div></div>"`;
```

Comment:
I dont see any helpertext in this snapshot

Reply:
or in the old one :/

Reply:
Looks like helperText takes a react node and not a return function. Seems to me this test should be removed or the component is incorrect. Should I add a comment or remove the test?